tests/client/actions/category.test.js

```diff
@@ -0,0 +1,29 @@
+import {
+  GET_CATEGORY_PENDING,
+  GET_CATEGORY_SUCCESS,
+  GET_CATEGORY_ERROR,
+  getCategoryPending,
+  getCategorySuccess,
+  getCategoryError
+} from '../../../client/actions/category'
+
+describe('category actions', () => {
+  it('GET_CATEGORY_PENDING works', () => {
```

Comment:
I think this should be `getCategoryPending() works` 😊, as we're testing the action creator - not the action itself.